packages/table/src/table.tsx

```diff
@@ -386,9 +386,14 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     public renderHotkeys() {
-        return <Hotkeys>
-            {this.maybeRenderCopyHotkey()}
-        </Hotkeys>;
+        let hotkeys = [this.maybeRenderCopyHotkey(), this.maybeRenderSelectAllHotkey()];
+        // remove null elements
+        hotkeys = hotkeys.filter((element) => element != null);
```

Comment:
just inline in JSX below and use undefined instead of null: `{hotkeys.filter((element) => element !== undefined)}`